docs/quickstart.rst

```diff
@@ -9,10 +9,80 @@ Install from PyPI with pip:
 
 .. code-block:: bash
 
-    pip install auth_matrix
+    pip install maykin-auth-matrix
 
+Requirements
+============
+
+This application makes use of the AbstractBaseUser model from django.
+Generally, you should be able to use this application with any Django project that uses the default User model.
+If you are using a custom User model, you will need to make sure that it is compatible with the AbstractBaseUser model.
+We made it compatible for the username and email fields, but you might need to make some changes to the code to make it work with your custom User model.
```

Comment:
this should mention that things will work out of the box if you inherit from `AbstractUser`, and otherwise you can customize things through the `username` and `email` properties on your custom user model.